src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -90,6 +95,7 @@ public class SubscriptionDocumentsFetcher
                     _threadAllocations = NativeMemory.CurrentThreadStats;
                     using (doc.Data)
                     {
+                        size += new Size(doc.Data.Size, SizeUnit.Megabytes);
```

Comment:
Do we need to consider tombstones here?

Reply:
tombstones? we are not processing them here?